DBCDumpHost/Controllers/DataController.cs

```diff
@@ -76,35 +65,18 @@ public async Task<DataTablesResult> Get(CancellationToken cancellationToken, str
             {
                 // GET, backwards compatibility for scripts/users using this
                 foreach (var get in Request.Query)
-                {
                     parameters.Add(get.Key, get.Value);
-                }
             }
 
-            if (!parameters.ContainsKey("search[value]"))
-            {
-                parameters.Add("search[value]", "");
-            }
-
-            var searchValue = parameters["search[value]"].Trim();
```

Comment:
Is the incoming search value still trimmed? Helps when copy pasting things that have whitespace around them.

Reply:
They are now! 😅